heat/core/dndarray.py

```diff
@@ -483,6 +484,131 @@ def astype(self, dtype, copy=True):
 
         return self
 
+    def balance_(self):
```

Comment:
Why is the function called `balance_` instead of just `balance`? The `b.balance()` call results in error because this method does not exist.

Reply:
its because this modifies the DNDarray itself and does not create a copy of it.